internal/engine/actions/actions.go

```diff
@@ -138,22 +138,82 @@ func (rae *RuleActionsEngine) processAction(
 	params engif.ActionsParams,
 	metadata *json.RawMessage,
 ) (json.RawMessage, error) {
+	logger := zerolog.Ctx(ctx).Debug().
+		Str("action", string(actionType)).
+		Str("cmd", string(cmd))
+
+	// Exit early in case we shouldn't do anything, i.e., cmd != ActionCmdOn or cmd != ActionCmdOff
```

Comment:
This comment doesn't quite line up with the code below, and the code below seems clear without it.  Remove the comment?

Reply:
good point, I'll remove it 👍